index.html

```diff
@@ -65,19 +65,85 @@ <h1>Hello, world!</h1>
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h1>Poems</h1>
```

Comment:
I love how you replaced the placeholder text with actual poem text

Reply:
Oh wow! Really nice work. Good line breaks and picture linking. I should include some of that too. Keep it up!

Reply:
I do like the poets years added to each title, the pictures add great character to each poets story.

Reply:
This is great. Way more personal and interesting for the User experience. An acutal poem and pictures! Great work.